graphql-types/src/types-post-processor.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
```

Comment:
I know this is a simplistic implementation, but I think it's better to start simple and only add complexity when we realize we need it.

Reply:
Updated here: https://github.com/commercetools/test-data/commit/9c903f870743350ce9cb61d227fb7e65c53c39b0